packages/apptive_grid_theme/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://www.apptivegrid.de
 repository: https://github.com/ApptiveGrid/apptive_grid_flutter/tree/main/packages/apptive_grid_theme
 
 environment:
-  sdk: '>=3.0.0 <4.0.0'
+  sdk: '>=2.17.0 <3.0.0'
```

Comment:
Guess the upper bound could still be adjusted to 4.0.0 (should be in a separate PR though in order for melos to version correctly)

Reply:
If its to be done in a different PR, can we merge this one?